include/mbedtls/ssl_internal.h

```diff
@@ -270,7 +270,15 @@ struct mbedtls_ssl_handshake_params
 #endif
 #if defined(MBEDTLS_ECDH_C)
     mbedtls_ecdh_context ecdh_ctx;              /*!<  ECDH key exchange       */
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_ecc_curve_t ecdh_psa_curve;
+    psa_key_slot_t ecdh_psa_privkey;
+    unsigned char ecdh_psa_peerkey[256];
+    size_t ecdh_psa_peerkey_len;
+#endif /* MBEDTLS_USE_PSA_CRYPTO */
 #endif
```

Comment:
Please add a 'MBEDTLS_ECDH_C' comment here, just for clarity.

Reply:
Yes.